src/_assets/stylesheets/main.scss

```diff
@@ -958,16 +958,17 @@ main .content {
 
 .breadcrumb {
   background-color: transparent;
+  padding: 0;
+  font-size: small;
```

Comment:
In this small font, the gray text seems too light and fuzzy. Let's make it a little darker.

(Alternatively, we could make the font size larger.)

Also, I happened to notice that when you select the gray text, it isn't legible. (Not enough contrast when the focus is on the window, and matches the color of the selection box when not focused.) Darkening the font a bit might fix that.

Reply:
Done. I've left it black.